src/modules/Workspaces/WorkspacesSnapshotTool/SnapshotUtils.cpp

```diff
@@ -191,6 +191,17 @@ namespace SnapshotUtils
         return res;
     }
 
+    bool IsFileManagerApp(std::wstring processPath)
+    {
+        std::wstring appName = std::filesystem::path(processPath).stem();
+        std::transform(appName.begin(), appName.end(), appName.begin(), towupper);
+        return ((appName == L"EXPLORER") // windows explorer
+                || (appName.starts_with(L"TOTALCMD")) // total commander
+                || (appName == L"DOPUS") // directory opus
+                || (appName == L"Q-DIR") // Q-Dir
+                || (appName.starts_with(L"XPLORER2"))); // Xplorer2
```

Comment:
```suggestion
return ((appName == NonLocalizable::FileManagers::FileExplorer)
        || (appName.starts_with(NonLocalizable::FileManagers::TotalCommander))
        || (appName == NonLocalizable::FileManagers::DirectoryOpus)
        || (appName == NonLocalizable::FileManagers::QDir)
        || (appName.starts_with(NonLocalizable::FileManagers::Xplorer2)));
```

Reply:
Together with adding constants into the `NonLocalizable` namespace 
```
  namespace FileManagers
  {
      const std::wstring FileExplorer = L"EXPLORER";
      const std::wstring TotalCommander = L"TOTALCMD";
      const std::wstring DirectoryOpus = L"DOPUS";
      const std::wstring QDir = L"Q-DIR";
      const std::wstring Xplorer2 = L"XPLORER2";
  }
```

Reply:
Done, thanks!